bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -131,7 +134,15 @@ object SemesterConverter {
 
   // definition of a ZeroConverter
   private case object ZeroSemesterConverter extends SemesterConverter {
-    override val transformers: List[TransformFunction] = Nil
+
+    val removeAustraliaSubmissions: TransformFunction = {
```

Comment:
This should likely be in `SemesterConverter2020BTo2021A`

Reply:
oh shit, i messed that one up somehow, thanks